rotkehlchen/tests/db/test_history_events.py

```diff
@@ -165,6 +188,17 @@ def test_read_write_events_from_db(database):
                     )
                 assert event == expected_event
 
+    with db.db.conn.read_ctx() as cursor:
+        for filter_query, entry_types, excluded_entry_types in entry_types_filter_query_args:
+            entry_types_to_include = HistoryEventSchema.get_entry_types_to_include(
+                entry_types, excluded_entry_types)
+            events = db.get_history_events(cursor, filter_query.make(entry_types=entry_types_to_include), True, False)  # noqa: E501
+            for event in events:
```

Comment:
I think it will make sense to check that the number of the events is the expected one.

If I am not mistaken, then currently, if `events` will be an empty list or some events will be missing, then nothing will fail.

Reply:
Yes indeed.